flink-end-to-end-tests/flink-end-to-end-tests-pulsar/src/test/java/org/apache/flink/tests/util/pulsar/PulsarSinkE2ECase.java

```diff
@@ -34,7 +34,7 @@
 /** Pulsar sink E2E test based on connector testing framework. */
 @SuppressWarnings("unused")
 @Tag("org.apache.flink.testutils.junit.FailsOnJava11")
-public class PulsarSinkE2ECase extends PulsarSinkTestSuiteBase {
+public class PulsarSinkE2ECase extends SinkTestSuiteBase<String> {
```

Comment:
Out of topic: IDEA hints me that this test class name doesn't match regex '[A-Z][A-Za-z\d]*Test(s|Case)?|Test[A-Z][A-Za-z\d]*|IT(.*)|(.*)IT(Case)?' . Do we run this test case properly?

Reply:
All the flink's e2e tests are in this naming suffix. I prefer to keep this.